gravitee-apim-console-webui/src/management/api/health-check-dashboard-v4/components/failed-health-checks/failed-health-checks.component.ts

```diff
@@ -0,0 +1,97 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, DestroyRef, OnInit } from '@angular/core';
+import { MatCardModule } from '@angular/material/card';
+import { GioLoaderModule } from '@gravitee/ui-particles-angular';
+import { ActivatedRoute } from '@angular/router';
+import { MatSort } from '@angular/material/sort';
+import { MatTableModule } from '@angular/material/table';
+import { AsyncPipe, DecimalPipe, TitleCasePipe } from '@angular/common';
+import { distinctUntilChanged, tap } from 'rxjs/operators';
+import { BehaviorSubject, combineLatestWith, Observable, switchMap } from 'rxjs';
+import { isEqual } from 'lodash';
+
+import { GioChartLineModule } from '../../../../../shared/components/gio-chart-line/gio-chart-line.module';
+import { ApiHealthV2Service } from '../../../../../services-ngx/api-health-v2.service';
+import { GioTableWrapperModule } from '../../../../../shared/components/gio-table-wrapper/gio-table-wrapper.module';
+import { GioTableWrapperFilters } from '../../../../../shared/components/gio-table-wrapper/gio-table-wrapper.component';
+import { SnackBarService } from '../../../../../services-ngx/snack-bar.service';
+import { HealthCheckLogsResponse } from '../../../../../entities/management-api-v2/api/v4/healthCheck';
+
+@Component({
+  imports: [
+    MatCardModule,
+    GioLoaderModule,
+    GioChartLineModule,
+    AsyncPipe,
+    GioTableWrapperModule,
+    MatSort,
+    MatTableModule,
+    DecimalPipe,
+    TitleCasePipe,
+  ],
+  selector: 'failed-health-checks',
+  standalone: true,
+  styleUrl: './failed-health-checks.component.scss',
+  templateUrl: './failed-health-checks.component.html',
+})
+export class FailedHealthChecksComponent implements OnInit {
+  defaultFilters: GioTableWrapperFilters = {
+    searchTerm: '',
+    pagination: { index: 1, size: 10 },
+  };
+
+  protected filters$ = new BehaviorSubject<GioTableWrapperFilters>(this.defaultFilters);
+
+  private apiId = this.activatedRoute.snapshot.params.apiId;
+  private showSuccessLogs = false;
+  public isLoading = true;
+
+  protected displayedColumns: string[] = ['timestamp', 'endpoint', 'gateway'];
+
+  protected logs$: Observable<HealthCheckLogsResponse>;
+
+  constructor(
+    private readonly activatedRoute: ActivatedRoute,
+    public readonly apiHealthV2Service: ApiHealthV2Service,
+    public readonly snackBarService: SnackBarService,
+    public readonly destroyRef: DestroyRef,
+  ) {}
+
+  ngOnInit() {
+    this.logs$ = this.apiHealthV2Service.activeFilter().pipe(
+      combineLatestWith(this.filters$),
+      distinctUntilChanged(isEqual),
+      switchMap(([timeRange, tableFilter]) =>
+        this.apiHealthV2Service.getApiHealthCheckLogs(this.apiId, {
```

Comment:
the loader on refresh / filter change do not gets the isLoading = true ; so the is no loading indicator in UI